edx_proctoring/static/proctoring/templates/student-onboarding-status.underscore

```diff
@@ -0,0 +1,109 @@
+                                <%= n %>
+                            </a>
+                        </li>
+                    <% } %>
+                    <% if (!nextPage){ %>
+                    <li class="disabled">
+                    <a aria-label="Next">
+                        <span aria-hidden="true">&raquo;</span>
+                      </a>
+                    </li>
+                    <% } else { %>
+                    <li>
+                    <a class="target-link" href="#" aria-label="Next" data-page-number="<%= currentPage + 1 %>"
+                    >
+                      <span aria-hidden="true">&raquo;</span>
+                    </a>
+                    </li>
+                    <% }%>
+                </ul>
+                <div class="clearfix"></div>
+            </div>
+            <div>
+                <ul class="status-checkboxes">
+                    <% _.each(onboardingStatuses, function(status){ %>
+                        <li>
+                            <input type="checkbox" id="<%= status %>" value="<%= status %>" 
+                            <% if (filters.includes(status)) { %>checked="true"<% } %>>
+                            <label for="<%= status %>">
+                                <%- interpolate(gettext(" %(onboardingStatus)s "), 
+                                { onboardingStatus: getOnboardingStatus(status) }, true) %>
+                            </label>
+                        </li>
+                    <% }); %>
+                </ul>
+            </div>
+            <table class="onboarding-status-table">
+                <thead>
+                    <tr class="onboarding-status-headings">
+                        <th class="username-heading"><%- gettext("Username") %></th>
+                        <th class="onboarding-status-heading"><%- gettext("Onboarding Status") %></th>
+                        <th class="last-updated-heading"><%- gettext("Last Modified") %> </th>
+                    </tr>
+                </thead>
+                <% if (isOnboardingItems) { %>
+                <tbody>
+                    <% _.each(onboardingItems, function(item){ %>
+                        <tr class="onboarding-items">
+                            <td>
+                                <%- interpolate(gettext(" %(username)s "), { username: item.username }, true) %>
+                            </td>
+                            <td>
+                                <%- interpolate(gettext(" %(onboardingStatus)s "), 
+                                { onboardingStatus: getOnboardingStatus(item.status) }, true) %>
+                            </td>
+                            <td><%= getDateFormat(item.modified) %></td>
+                        </tr>
+                    <% }); %>
+                </tbody>
+                <% } %>
+            </table>
+            <% if (!isOnboardingItems) { %>
+                <p>No onboarding data found.</p>
```

Comment:
Could you please add more styles to this case? Make the margin bigger and make the text size bigger and bolder.
Also, could you be more specific about onboarding data? I believe we can only see this message if the course have no enrollment fits the onboarding scenario right? How about update the text to be
`There are no users enrolled into this course that requires onboarding exams`